app/utils/s3.py

```diff
@@ -23,7 +24,7 @@ def __init__(  # pylint: disable=too-many-arguments
         verify,
         cert,
         private_key,
-    ):
+    ):  # pylint: disable=too-many-arguments
```

Comment:
even here I'm curious: why would pylint complain about too many arguments, and why do we want to have many arguments here anyway?

Reply:
Having too many arguments can mean there's a higher risk of switching up the order (and it's more difficult to spot). I'm not sure how pylint considers it, but one common solution for python is to use keyword arguments instead.

The "requested" arguments are those that are part of the "complete" specification of a s3 resource, so they'll be needed to support a generic s3 resource.